GraphWorker.cpp

```diff
@@ -0,0 +1,78 @@
+#include "include/GraphWorker.h"
+#include "include/graph.h"
+#include <buffer_tree.h>
+
+#include <fstream>
+#include <string>
+
+bool GraphWorker::shutdown = false;
+int GraphWorker::num_groups = 1;
+int GraphWorker::group_size = 1;
+const char *GraphWorker::config_file = "graph_worker.conf";
+GraphWorker **GraphWorker::workers;
+
+void GraphWorker::startWorkers(Graph *_graph, BufferTree *_bf) {
+	std::string line;
+	std::ifstream conf(config_file);
+	if (conf.is_open()) {
+		while(getline(conf, line)) {
+			if(line.substr(0, line.find('=')) == "num_groups") {
+				num_groups = std::stoi(line.substr(line.find('=')+1));
+				printf("Number of groups = %i\n", num_groups);
+			}
+			if(line.substr(0, line.find('=')) == "group_size") {
+				group_size = std::stoi(line.substr(line.find('=')+1));
+				printf("Size of groups = %i\n", group_size);
+			}
+		}
+			
+	} else {
+		printf("WARNING: Could not open thread configuration file!\n");
```

Comment:
Add default configuration notification

Reply:
What do you mean? you want me to change the text?

Reply:
Yes, since we default to 1 and 1 our users (probably me) should be notified of that.